libraries/redshop/helper/product.php

```diff
@@ -5133,5 +5136,39 @@ public static function checkCompareProduct($productId)
         return '';
     }
 
+    /**
+     * @param   integer    $productId     Product ID
+     * @param   float    $productPrice  Product price
+     *
+     * @return  float
+     *
+     */
+    public function calculateProductApplyMassDiscount($productId, $productPrice)
```

Comment:
@hoangvan149dhv Why we need pass 2nd param `$productPrice? Is it modified price already or the origin product price? If origin we can get it from Product Entity when we has productId